openpgp/keys_test.go

```diff
@@ -3,6 +3,7 @@ package openpgp
 import (
 	"bytes"
 	"crypto"
+	_ "crypto/md5"
```

Comment:
AFAIU, this only fixes it when running the test - not when using the package outside of tests, as in that case this file is not imported, and hence neither is `crypto/md5`. So it's probably better to import it in `openpgp/internal/algorithm/hash.go`.